desktop/mac/release-notes/index.md

```diff
@@ -17,6 +17,38 @@ redirect_from:
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 4.2.0
+2021-11-xx
+
+> Download Docker Desktop
+>
+> [Mac with Intel chip](??: .button .primary-btn }
+> [Mac with Apple chip](??: .button .primary-btn }
```

Comment:
I think here it's missing the link isn't it?

Reply:
Yes, these lines should be copied from lines 57-58 (from 4.1.1) to have the correct links for the latest release.
And then lines 57-58 need to be updated like lines 73-74 without UTM, but with the build number for 4.1.1.